torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -216,6 +218,42 @@ def get_default_x86_inductor_quantization_config(is_qat: bool = False):
     return quantization_config
 
 
+@functools.lru_cache
+def get_default_x86_inductor_dynamic_quantization_config():
+    act_quantization_spec = QuantizationSpec(
+        dtype=torch.uint8,
+        quant_min=0,
+        quant_max=255,
+        is_dynamic=True,
+        observer_or_fake_quant_ctr=PlaceholderObserver.with_args(is_dynamic=True),
+    )
+
+    extra_args: Dict[str, Any] = {"eps": 2**-12}
+    weight_quantization_spec = QuantizationSpec(
+        dtype=torch.int8,
+        quant_min=-128,
+        quant_max=127,
+        qscheme=torch.per_channel_symmetric,
+        ch_axis=0,  # 0 corresponding to output channel
+        is_dynamic=False,  # Weight is not quantized at runtime but ahead of time
+        observer_or_fake_quant_ctr=PerChannelMinMaxObserver.with_args(**extra_args),
+    )
+    bias_observer_or_fake_quant_ctr: _ObserverOrFakeQuantizeConstructor = (
+        PlaceholderObserver
+    )
+    bias_quantization_spec = QuantizationSpec(
```

Comment:
this can be None, see https://github.com/pytorch/pytorch/blob/main/torch/ao/quantization/quantizer/xnnpack_quantizer.py#L166

Reply:
Thanks. I have changed it to None.